internal/note/note_verifier.go

```diff
@@ -71,25 +73,44 @@ func (v *verifier) Verify(msg, sig []byte) bool {
 	return v.v(msg, sig)
 }
 
-// NewSigstoreECDSAVerifier creates a new note verifier for checking ECDSA signatures over SHA256 digests.
+// NewECDSAVerifier creates a new note verifier for checking ECDSA signatures over SHA256 digests.
 // This implementation is compatible with the signature scheme used by the Sigstore Rékor Log.
 //
 // The key is expected to be provided as a string in the following form:
-//   <key name> " " <Base64 encoded SubjectPublicKeyInfo DER>
+//   <key_name>+<key_hash>+<key_bytes>
+// Where
+//   <key_name> is a human readable identifier for the key, containing no whitespace or "+" symbols
+//   <key_hash> is a 32bit hash of the key bytes
```

Comment:
32bit hash: is this more precisely the first 32 bits of a SHA256 over key_bytes?

Reply:
Yep, that is technically what the implementation is, but I was being intentionally vague here to line up with the proposed change to the comments in note: https://go-review.googlesource.com/c/mod/+/364856 (e.g. folks shouldn't _rely_ on this being a truncated SHA256 over the DER).

I've tweaked the comment slightly though to reflect that it's the DER and not the `key_bytes`.